myscripts.js

```diff
@@ -0,0 +1,9 @@
+window.addEventListener('load',()=>{
+    const navLink=document.querySelectorAll(".navLink");
+    const emptiness=document.querySelector(".empty_area");
+    navLink.forEach(name=> {
+        name.addEventListener('click',()=>{
+            emptiness.innerHTML=navLink.value;
```

Comment:
Why are you using the **value** of **_navLink_** here?

Please check to be using correct element